### PR TITLE
Backend Issues, Issue Assignees and Issue Comments are  implemented

### PR DESCRIPTION
**Pull request by cakmadam98**
All unit tests are passed. Documentation is also implemented.

### CODE REVIEW

File: Platon/backend/app/workspace_system/views.py [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+                if contributionSearch is None:
+                    return make_response(jsonify({'error': 'User is not a contributor'}), 401)
+
+                if not contributionSearch.is_active:
+                    return make_response(jsonify({'error': 'User is not an active contributor currently'}), 401)
+            
+            try:
+                issueAssigneeSearch = IssueAssignee.query.filter(IssueAssignee.issue_id == form.issue_id.data)
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
```
Comment (HUmutOzdemir) [inline Platon/backend/app/workspace_system/views.py:360]: Same as above.

**Review by HUmutOzdemir — CHANGES_REQUESTED** [review]

File: Platon/backend/app/workspace_system/views.py [hunk 2]
```diff
@@ -0,0 +1,577 @@
...
+                if contributionSearch is None:
+                    return make_response(jsonify({'error': 'User is not a contributor'}), 401)
+
+                if not contributionSearch.is_active:
+                    return make_response(jsonify({'error': 'User is not an active contributor currently'}), 401)
+            
+            try:
+                issueSearch = Issue.query.filter(Issue.workspace_id == workspace_id)
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
```
[the remark this answers is not in the document]
Comment (cakmadam98) [reply Platon/backend/app/workspace_system/views.py]: With the latest update, I've added all()

File: Platon/backend/app/workspace_system/views.py (hunk 1 above)
Comment (cakmadam98) [reply Platon/backend/app/workspace_system/views.py]: With the latest update, I've added all()

File: Platon/backend/test.py [hunk 3]
```diff
@@ -3,7 +3,8 @@
 #from tests.auth_system_test import LoginTest,ResetPasswordTest
 #from tests.follow_test import FollowTest
 #from tests.profile_management_test import ResearchInfoTests,NotificationTests
-from tests.search_engine_test import SearchHistoryTests, UserSearchTests
+#from tests.search_engine_test import SearchHistoryTests, UserSearchTests
```
[the remark this answers is not in the document]
Comment (cakmadam98) [reply Platon/backend/test.py]: To test my changes quickly, I've commented them. I've forgot them when I push here. Now they are all active.

**Review by HUmutOzdemir — APPROVED** [review]

**Pull request merged**